types/fluent-ffmpeg/index.d.ts

```diff
@@ -296,9 +297,12 @@ declare namespace Ffmpeg {
         mergeToFile(target: string | stream.Writable, options?: { end?: boolean }): FfmpegCommand;
         concatenate(target: string | stream.Writable, options?: { end?: boolean }): FfmpegCommand;
         concat(target: string | stream.Writable, options?: { end?: boolean }): FfmpegCommand;
+        clone(): FfmpegCommand;
+        run(): void;
     }
 }
 declare function Ffmpeg(options?: Ffmpeg.FfmpegCommandOptions): Ffmpeg.FfmpegCommand;
 declare function Ffmpeg(input?: string | stream.Readable, options?: Ffmpeg.FfmpegCommandOptions): Ffmpeg.FfmpegCommand;
 
+export as namespace Ffmpeg;
```

Comment:
If that line is not included I am unable to get any types from the definition. Lib is used in node.js app and is imported by

`import ffmpeg = require('fluent-ffmpeg')`

If you have a better suggestion I can modify.